.changeset/unlucky-plants-give.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-components': patch
```

Comment:
But I'm unsure on specific backstage policy.

Reply:
@backstage/maintainers, I would like to hear your recommendation too.

Reply:
For reference,

https://backstage.io/docs/overview/versioning-policy/

If I understand correctly, because `@backstage/core-components` has not yet hit v1.0.0, project policy does not need to support full semver yet, though that is really in re: to breaking changes. I'd agree with @aitherios that since this is a new feature, it would make sense (to me) to minor bump it :)

https://github.com/backstage/backstage/blob/c236193f513c70230be732eb3d424a2ff9233770/packages/core-components/package.json#L4

Let's see what the maintainer friends say!

Reply:
It'd be a `patch`, no breaking changes and it's still on 0.x. It's compatible with semver but not explicitly required by semver, since semver takes a pretty loose approach to v0.

Reply:
Thanks everybody, I'll keep it as patch!